uxbackend/uxbackend.go

```diff
@@ -0,0 +1,200 @@
+package uxbackend
```

Comment:
There is no need to create an seperate pkg for this, You can create a file under controllers.